central/compliance/service/service_impl.go

```diff
@@ -127,9 +127,16 @@ func (s *serviceImpl) GetAggregatedResults(ctx context.Context, request *v1.Comp
 		return nil, err
 	}
 
+	filteredSources := sources[:0]
+	for _, source := range sources {
+		config, exsist, _ := s.complianceDataStore.GetConfig(ctx, source.GetStandardId())
```

Comment:
I looked at other usages and let's make sure we are handling the errors that may come back from this datastore appropriately.